xmtp_mls/src/groups/group_permissions.rs

```diff
@@ -994,7 +1054,7 @@ pub(crate) fn policy_all_members() -> PolicySet {
     }
     PolicySet::new(
         MembershipPolicies::allow(),
-        MembershipPolicies::allow(),
+        MembershipPolicies::allow_if_actor_admin(),
```

Comment:
Remove members should require admin /super admin to be aligned with Permissions XIP. Fixes https://github.com/xmtp/libxmtp/issues/857